kmsauth/__init__.py

```diff
@@ -19,6 +19,22 @@
 PY2 = sys.version[0] == '2'
 
 
+def ensure_str(str_or_bytes, encoding='utf-8'):
```

Comment:
- `ensure_text` is a better name for this
- why not `errors` parameter here as well?